message_center_compassion/mappings/compassion_mapping.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2015 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Flückiger Nathan <nathan.fluckiger@hotmail.ch>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+from odoo import models, fields, api
+
+
+class CompassionMapping(models.Model):
+
+    _name = "compassion_mapping"
+
+    name = fields.Char()
+    # Models and fieldToJson link
+    fields_json_ids = fields.Many2many('field_to_json')
+
```

Comment:
Shouldn't we add a `model_id` relation field in order to indicate to which model the mapping refers to? It would be:
`model_id = fields.Many2one('res.model', 'Model')`